feed/views.py

```diff
@@ -1,9 +1,21 @@
 from django.shortcuts import render
 from .models import Post
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.views.generic import CreateView
 
 
 def feed(request):
     context = {
         'posts': Post.posts.main_feed()
     }
     return render(request, 'feed/feed.html', context)
+
+
+class PostCreateView(LoginRequiredMixin, CreateView):
```

Comment:
Suggest to follow documentation
https://docs.djangoproject.com/en/4.0/topics/auth/default/#how-to-log-a-user-in-1

Reply:
Im not sure what to change here in `PostCreateView` but I've applied the suggestions to the `test_authenticated_users_access` test.
Would be happy for more guidance

Reply:
Maybe this will give you one tests idea
https://github.com/redhat-beyond/beyond-tutorial/blob/main/msgboard/tests.py#L25-L29

And this 
https://github.com/redhat-beyond/beyond-tutorial/blob/main/msgboard/tests.py#L47-L51

Look at the whole login under views
https://github.com/redhat-beyond/beyond-tutorial/blob/main/msgboard/views.py#L7

Suggested to work as per the docs, Stupid Simple

Reply:
Yes, but NVM. there is different way to do that